File: CNC modeling 2.0/src/com/sergey/pisarev/model/Point.java
package com.sergey.pisarev.model;

public class Point {


    private float x;

    private float z;

    public Point() {

    }

    public Point(float x, float z) {
        this.x = x;
        this.z = z;
    }

    public float getX() {
        return x;
    }

    public void setX(float x) {
        this.x = x;
    }

    public float getZ() {
        return z;
    }

    public void setZ(float z) {
        this.z = z;
    }
}

File: CNC modeling 2.0/src/main/java/com/sergey/pisarev/model/Point.kt
package com.sergey.pisarev.model

open class Point : Text {

    open var x = N_GANTRYPOS_X
    open var z = N_GANTRYPOS_Z

    constructor() {}
    constructor(x: Double, z: Double) {
        this.x = x
        this.z = z
    }
}